pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -838,6 +838,8 @@ type StorageSpec struct {
 	// The easiest way to use a volume that cannot be automatically provisioned
 	// is to use a label selector alongside manually created PersistentVolumes.
 	VolumeClaimTemplate EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
+
```

Comment:
Add a comment explaining what this field does, probably same as the previous one (without the deprecation notice) but plural.